content/blog/2016/2016-05-26-pipeline-snippetizer.adoc

```diff
@@ -0,0 +1,26 @@
+---
+layout: post
+title: New display of Pipeline’s “snippet generator”
```

Comment:
Umm, Emacs? Yes use of the typographically correct symbols was intentional. Why would you want to use ASCII quotes?